plugins/nextcloud-rule-exclusions-before.conf

```
@@ -799,6 +799,37 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/(?:[^/]+/){2}[
         "t:none,\
         ctl:ruleRemoveById=950100"
 
+# Deleting a contact from addressbook
+# XML parser fails because a content type is set without a request body
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/[^/]+/[^/]+\.vcf$" \
```

This seems easier to read to me. But keep it how you like.
```suggestion
SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users(?:/[^/]+){3}\.vcf$" \
```